src/Orchard.Web/Themes/TheAdmin/Scripts/admin.js

```diff
@@ -93,18 +93,10 @@
 
         form.addClass(submittingClass);
 
-        buttons = form.find("[type='submit']");
-        buttons.prop("disabled", true)
-
         // safety-nest in case the form didn't refresh the page
         setTimeout(function () {
             form.removeClass(submittingClass);
-            buttons.prop("disabled", false)
```

Comment:
Just using this PR to comment on my code, I think this would be better to use a closure with the previous state of `disabled` in case is was not `false`.